module-federation/module-federation-examples
# Fail to render page when using angular material

I was recently trying to play with webpack module federation and angular mfe's that contain Angular CDK/Material.

How to reproduce the issue:
1. start both microfrontends
2. open the shell and simple switch between the Home and Profile

You will notice that the home page keeps adding the Profile page, without refreshing.
